python/chainer_compiler_core.cc

```diff
@@ -95,10 +97,24 @@ std::shared_ptr<runtime::XCVM> Compile(
     return std::make_shared<runtime::XCVM>(xcvm_prog);
 }
 
+bool IsParam(Value* value) {
+    const std::string& name = value->name();
+    // the second condition is for ch2o
```

Comment:
This is fine, but I'd like to see the check for `/` in Python side. Anyway, could you add "TODO(hamaji): Remove the check for '/' after deprecating CH2O" or something?